.circleci/config.yml

```diff
@@ -29,12 +29,11 @@ _refs:
     command: yarn build
   test: &test
     name: Testing
-    command: yarn ci-test
+    command: yarn test
   coverage: &coverage
     name: Submitting code coverage to codecov
     command: |
-      ./node_modules/.bin/nyc report --reporter text-lcov > coverage.lcov
-      curl -s https://codecov.io/bash | bash
+      # yarn codecov
```

Comment:
I can't get codecov working. It has actually never been working.. so I will look at this later.